packages/amplify-appsync-simulator/src/server/operations.ts

```diff
@@ -33,54 +30,29 @@ export class OperationServer {
     private subscriptionServer: SubscriptionServer,
   ) {
     this.port = config.port;
-    this.app = express();
-    this.app.use(express.json({ limit: MAX_BODY_SIZE }));
-    this.app.use(cors());
-    this.app.post('/graphql', this.handleRequest.bind(this));
-    this.app.get('/api-config', this.handleAPIInfoRequest.bind(this));
-    this.app.use('/', express.static(STATIC_ROOT));
+    this._app = express();
+    this._app.use(express.json({ limit: MAX_BODY_SIZE }));
+    this._app.use(cors());
+    this._app.post('/graphql', this.handleRequest.bind(this));
```

Comment:
we could make handlerequest and the other method arrow functions so no old school bind required.